package.json

```diff
@@ -1,5 +1,6 @@
 {
   "scripts": {
+    "prebuild": "lerna run build --scope @redwoodjs/router",
```

Comment:
@peterp How can we do this in a nicer way?

Reply:
Could you let me know what this is solving? Do you need the typings for another package?

Reply:
Is it the typing that you require?

Reply:
Without it I get this
![image](https://user-images.githubusercontent.com/30793/107083071-62f7fd80-67f5-11eb-9bec-078bdb9b59bb.png)

Reply:
Testing needs stuff from router, but it's building testing first.

If I run the build twice it'll work the second time because then it has the typings from the previous build

Reply:
Solved it! I'll update the PR